Tests/Rules/AvoidDefaultValueForMandatoryParameter.tests.ps1

```diff
@@ -21,5 +27,11 @@ Describe "AvoidDefaultValueForMandatoryParameter" {
                 Where-Object { $_.RuleName -eq $ruleName }
             $violations.Count | Should -Be 0
         }
+
+        It "returns no violations when the parameter is specified as mandatory=0 and has a default value" {
+            $violations = Invoke-ScriptAnalyzer -ScriptDefinition 'Function foo{ Param([Parameter(Mandatory=$false)]$Param1=''val1'', [Parameter(Mandatory)]$Param2=''val2'', $Param3=''val3'') }' |
+                Where-Object { $_.RuleName -eq $ruleName }
```

Comment:
This test seems to be a copy-paste version of the test above if I am not mistaking? I think you meant to change it t have `Mandatory = 0`. A cleanup of the other unnecessary and confusing parameters `$Param2` and `$Param3` .

Reply:
Fixed.